src/test/datascience/interactiveDebugging.vscode.test.ts

```diff
@@ -47,6 +48,7 @@ suite('Interactive window debugging', async function () {
         traceInfo(`Start Test ${this.currentTest?.title}`);
         api = await initialize();
         disposables.push(vscode.debug.registerDebugAdapterTrackerFactory('python', tracker));
+        disposables.push(vscode.debug.registerDebugAdapterTrackerFactory(pythonIWKernelDebugAdapter, tracker));
```

Comment:
Don's change for IW debugging uses a different debug type than 'python', so we need to register a tracker for his new type here in order to watch the debugger in the tests. Depends upon version of ipython though so I left the other tracker there too.

Reply:
The IW tests loop through the two debugger types for the entire suite. Was that supposed to happen in here instead maybe?